resources/public/api/conf/2.0/scenarios/broughtForwardLosses_list.md

```diff
@@ -9,19 +9,27 @@
     <tbody>
         <tr>
             <td><p>N/A - DEFAULT</p></td>
-            <td><p>Simulate list containing 4 losses.</p></td>
+            <td><p>Simulates a list containing 4 losses.</p></td>
         </tr>
         <tr>
             <td><p>SELF_EMPLOYMENT</p></td>
-            <td><p>Simulate list containing two self-employment losses.</p></td>
+            <td><p>Simulates a list containing two self-employment losses.</p></td>
         </tr>
         <tr>
             <td><p>UK_PROPERTY_FHL</p></td>
-            <td><p>Simulate list containing single UK FHL property loss.</p></td>
+            <td><p>Simulates a list containing a single UK FHL property loss.</p></td>
         </tr>
         <tr>
             <td><p>UK_PROPERTY_NON_FHL</p></td>
-            <td><p>Simulate list containing single UK Non-FHL property loss.</p></td>
+            <td><p>Simulates a list containing a single UK Non-FHL property loss.</p></td>
+        </tr>
+        <tr>
+            <td>FOREIGN_PROPERTY</td>
```

Comment:
you are missing the paragraph tags <p> from the new entries hence it doesn't render correctly.